api/app/models/verse.rb

```diff
@@ -64,8 +64,9 @@ def active_tick!
       next_hidden_word.visible! 
       VerseTickJob.set(wait: (0.5).seconds).perform_later(self.uuid)
     else
-      VerseTickJob.set(wait: (8).seconds).perform_later(self.uuid)
+      sleep 2.5
```

Comment:
@dinushkaherath we can't use sleep on the rails server because it just ties up the thread, gotta figure out how to delay things with perform_later

Reply:
oooo